android-components/components/feature/search/src/main/assets/search/list.json

```diff
@@ -126,14 +126,14 @@
     "ca": {
       "default": {
         "visibleDefaultEngines": [
-          "google-b-m", "ddg", "wikipedia-ca"
+          "google-b-m", "bing", "ddg", "ebay-es", "wikipedia-ca"
         ]
       }
     },
     "cak": {
       "default": {
         "visibleDefaultEngines": [
-          "google-b-m", "amazondotcom", "ddg", "wikipedia-es"
+          "google-b-m", "bing", "amazondotcom", "ddg", "wikipedia-es"
```

Comment:
I didn't see this mentioned in https://bugzilla.mozilla.org/show_bug.cgi?id=1818321#c5, but I do see it on the spreadsheet. Just wondering if this is a desired change and how we arrived at adding it despite not being on the list mentioned by @Standard8.